go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/canonical/lxd
 go 1.22.5
 
 require (
+	github.com/NVIDIA/nvidia-container-toolkit v1.16.1
```

Comment:
please can this commit message be clearer as to what it is doing - it is adding a new dependency for github.com/NVIDIA/nvidia-container-toolkit

Reply:
the commit title still needs to be changed to "add a new dependency for github.com/NVIDIA/nvidia-container-toolkit" otherwise it'll be difficult to see what this does when backporting.

Reply:
fixed